x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -46,6 +56,11 @@ export const PolicyList = memo(() => {
     customQueryOptions: { enabled: policyIds.length > 0 },
   });
 
+  // grab endpoint version for empty page
+  const { data: endpointPackageInfo } = useGetEndpointSecurityPackage({
```

Comment:
Should we get the isLoading/isRefetching stats here and use those in the loading state?

Reply:
Also, what if there is an error on this api call? Should we display a toaster message?

Reply:
i'll grab the isFetching for this call and put it into the loading prop for the `PolicyEmptyPage`. Do we usually display a toaster message if grabbing the endpoint package fails? We only need the endpoint package for the `PolicyEmptyState` so that we can display the correct link to fleet to add a new policy. It doesn't affect anything besides that.

Reply:
I would say if any http request that affects UI fails, let's show an error message, otherwise UI can stop working as expected and user won't know why.